EveripediaNetwork/ep-ui
# Link redirects

# Fix incorrect link for Suchet Dhindsa's wiki page

_Fixes the issue where clicking on Suchet Dhindsa's wiki page on the about page takes you to Justin Moore's wiki page. The link has been updated to the correct URL._

## How should this be tested?

1. Navigate to the about page.
2. Click on Suchet Dhindsa's wiki page link.
3. Verify that it redirects to the correct URL: https://iq.wiki/wiki/suchet-dhindsa-salvesen/

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/3061
